unlock-app/src/components/content/LogContent.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import Head from 'next/head'
+import Layout from '../interface/Layout'
+import CreatorAccount from '../creator/CreatorAccount'
+import CreatorLog from '../creator/CreatorLog'
+import BrowserOnly from '../helpers/BrowserOnly'
+import GlobalErrorConsumer from '../interface/GlobalErrorConsumer'
+import { pageTitle } from '../../constants'
+import withConfig from '../../utils/withConfig'
+import * as UnlockTypes from '../../unlock'
+
+interface Props {
+  account: UnlockTypes.Account
+  network: UnlockTypes.Network
+  transactionFeed: UnlockTypes.Transaction[]
+  transactionMetadataMap: UnlockTypes.TransactionMetadataMap
+}
+
+export const LogContent = (props: Props) => {
+  const { account, network, transactionFeed, transactionMetadataMap } = props
+  return (
+    <GlobalErrorConsumer>
+      <Layout title="Creator Log">
+        <Head>
+          <title>{pageTitle('Log')}</title>
+        </Head>
+        {account && (
+          <BrowserOnly>
+            <CreatorAccount network={network} account={account} />
+            <CreatorLog
+              transactionFeed={transactionFeed}
+              transactionMetadata={transactionMetadataMap}
+            />
+          </BrowserOnly>
+        )}
+      </Layout>
+    </GlobalErrorConsumer>
+  )
+}
+
+export const readable = (type: UnlockTypes.TransactionType) => {
```

Comment:
Could you please add a TODO to eventually use a better mechanism (string mapping!) because I am pretty sure in the long term this will come and bite us!

Reply:
My preferred solution for this would be to just change the enum values to the readable forms. Insofar as I'm aware, we never actually use those bare string values for anything but this.